rero/rero-ils-ui
# fix tests

__admin records: backport briev views and editor__

* Adds the brief views for all the resources.
* Configures the menu.
* Adds routes for the record resources.
* Backports the library and circulation policies editors.
* Fixes lint for all the backport components.

__admin layout: implement layout and tools__

* Adds a footer.
* Adds a new menu component for navigation.
* Adds a front page.
* Adds a user service to retrieve the logged user informations.
* Updates the app-config service to retrieve the backend settings.

Co-Authored-by: Johnny Mariéthoz <Johnny.Mariethoz@rero.ch>
Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

**Note:** this PR does not pass the tests, I suspect that it is related to `npm link` and needs also some additional job for the custom editors. Moreover, it contains multiple commits, thus please check the commits individually.

### To Test

- clone this project run `npm i`
- run `npm run start-proxy` and check the interface
- run `./run-tests.sh`
